src/app/assets/css/maevsi.css

```
@@ -49,25 +49,55 @@
     }
   }
 
+  --default-font-family: var(--font-manrope);
   --font-manrope:
     'Manrope Variable', ui-sans-serif, system-ui, sans-serif,
     'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
-  --default-font-family: var(--font-manrope);
+  --color-accent-fancy: #4743b1;
+  --color-background-bright: var(
+    --color-white
+  ); /* TODO: replace with new design variables */
+
+  --color-background-brighten: var(
+    --color-zinc-200
+  ); /* TODO: replace with new design variables */
+
+  --color-background-card-bright: #fafafc;
```

Please use the style names from Figma for all new variables in this file, e.g.

```suggestion
  --color-base-white: #fafafc;
```

---

I've made this change as an example and pushed it here, but please align the other variable names.